index.html

```diff
@@ -0,0 +1,171 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>VidiTube</title>
+    <link href="stylesheet.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="header-section">
+        <header>
+            <div id="logo">
+                <img src="./images/logo.PNG" alt="VidiTube Logo" width="60" height="60" class="image-logo"/>
+                <span class="name">VidiTube</span>
+            </div>
+
+            <form class="search-bar">
+               <input class="search-input" type="search" placeholder="Search" aria-label="Search"/>
+               <button type="submit" class="search-btn">
+                    <img src="./images/search_icon.svg" alt="search"/>
+               </button>
```

Comment:
- [ ] Please remember your input should have the `require` attribute for validation purposes so that a user can not submit empty input